src/test/java/XmlParserTest.java

```diff
@@ -0,0 +1,20 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class XmlParserTest {
+
+    @Test
+    void should_should_return_users_from_users_xml() {
+        // given
+        String file = "users.xml";
```

Comment:
To make the test successful, most likely this file should be in test resources, but you need to check it